Stringtable.xml tidyup

DrunkOnSuccess opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [ ] Enhancement
4. [X] Miscellaneous

### What have you changed and why?
Information: cleaned up a lot of the text, mostly Socrates' stuff, it should hopefully erase most of the broken English that was present in the mod. Also cleaned up some vague mission text and made it easier to understand some mission objectives.

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: start missions, read their descriptions, imprison counter-insurgents, call in supports.

********************************************************
Notes: most text changed was to do with missions, counter-insurgency dialogue and some dialogue in the commander menu. I wasn't able to test each and every one thoroughly but did do multiple passes of the .xml files. I would recommend further checks in-game but everything I did manage to check worked perfect.

DrunkOnSuccess committed:
Stringtable.xml tidyup

DrunkOnSuccess committed:
Update Stringtable.xml

Psycool3695 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2709,7 +2709,7 @@
             </Key>
             <Key ID="STR_hints_new_game_petros_dead_text">
-                <Original>%1 has been killed. You lost part of your assets and need to select a new HQ position far from the enemies.</Original>
+                <Original>%1 has been killed. You lost mostt of your assets and need to select a new HQ position far from enemy presence.</Original>
```
A typo confirmed.

DrunkOnSuccess replied:
Just spotted that myself, whoops.

DrunkOnSuccess replied:
Resolved in new update to the file.

DrunkOnSuccess committed:
Update 2

Changed some typos and corrected some oversights in the text

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -889,7 +889,7 @@
             </Key>
             <Key ID="STR_A3AP_rally_tow_rope">
-                <Original>You can't travel with your Tow Rope out or a Vehicle attached.</Original>
+                <Original>You can't travel with your tow rope out or with a vehicle attached.</Original>
```
```suggestion
                <Original>You can't travel with your tow rope out or with a vehicle in tow.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -1347,7 +1347,7 @@
             </Key>
             <Key ID="STR_comms_mp_loot_heli">
-                <Original>&lt;t size='0.6'&gt;%1 helicopter and its crew helps us with the scattered &lt;t size='0.6' color='#660080'&gt;loot&lt;/t&gt;. They will drop the crate with gathered loot near your position.&lt;/t&gt;</Original>
+                <Original>&lt;t size='0.6'&gt;%1 The helicopter and its crew helps us with the scattered &lt;t size='0.6' color='#660080'&gt;loot&lt;/t&gt;. They will drop the crate with gathered loot near your position.&lt;/t&gt;</Original>
```
I'd revert this change.

%1 is a variable substitution for the name of the rebel faction. E.g. the final string when playing as FIA will read "FIA The helicopter and its crew helps us ..." with this change.

DrunkOnSuccess replied:
I feel it would be better changed to remove that %1 and have it be "Our helicopter and its crew" as "FIA helicopter and its crew" is a bit awkward to read and it's unnecessary to specify the rebel faction when we are obviously the one that calls it in anyways.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -1503,7 +1503,7 @@
             </Key>
             <Key ID="STR_end_mission_modUnautorized_desc">
-                <Original>An incompatible mod or incompatible set of supported mods installed on the server or your PC has been detected. To avoid problems the mission has ended. Please turn off unsupported (ASR AI, aLIVE, MCC or any AI behaviour) mods from your computer (and server when applicable) to be able to play Antistasi. See server .rpt log for more information.</Original>
+                <Original>An incompatible mod or incompatible set of supported mods installed on the server or your PC has been detected. To avoid problems the mission has ended. Please turn off unsupported (ASR AI, aLIVE, MCC, LAMBS or any other AI behaviour) mods from your computer (and server when applicable) to be able to play Antistasi. See server .rpt log for more information.</Original>
```
```suggestion
                <Original>An incompatible mod or incompatible set of supported mods installed on the server or your PC has been detected. To avoid problems the mission has ended. Please turn off unsupported (ASR AI, aLIVE, MCC, LAMBS_Danger.fsm or any other AI behaviour) mods from your computer (and server when applicable) to be able to play Antistasi. See server .rpt log for more information.</Original>
```

*Technically,* LAMBS danger is disabled anyway so it shouldn't really matter (or so I was told). The other LAMBS mods are fine.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_AS_hint_text">
-                <Original>Assassination Missions require cities, patrolled forests, open plains or airports closer than %1m from your HQ.</Original>
+                <Original>Assassination missions require cities, patrolled forests, open plains or airports closer than %1m from your HQ.</Original>
```
```suggestion
                <Original>Assassination missions require cities, patrolled forests, open plains or airports within %1m of your HQ.</Original>
```

subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_CON_hint_text">
-                <Original>Conquest Missions require roadblocks or outposts closer than %1m from your HQ.</Original>
+                <Original>Conquest missions require roadblocks or outposts closer than %1m from your HQ.</Original>
```
```suggestion
                <Original>Conquest missions require roadblocks or outposts within %1m of your HQ.</Original>
```
subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_DES_hint_text">
-                <Original>Destroy Missions require Airbases or Radio Towers closer than %1 from your HQ.</Original>
+                <Original>Destroy missions require Airbases or Radio Towers closer than %1 from your HQ.</Original>
```
```suggestion
                <Original>Destroy missions require Airbases or Radio Towers within %1 of your HQ.</Original>
```
subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_LOG_hint_text">
-                <Original>Logistics Missions require Outposts, Seaports or Banks closer than %1m from your HQ.</Original>
+                <Original>Logistics missions require Outposts, Seaports or Banks closer than %1m from your HQ.</Original>
```
```suggestion
                <Original>Logistics missions require Outposts, Seaports or Banks within %1m of your HQ.</Original>
```
subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_SUPP_hint_text">
-                <Original>Support Missions require Cities closer than %1m from your HQ.</Original>
+                <Original>Support missions require cities closer than %1m from your HQ.</Original>
```
```suggestion
                <Original>Support missions require cities within %1m of your HQ.</Original>
```
subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2329,56 +2329,56 @@
             </Key>
             <Key ID="STR_chats_mission_request_no_RES_hint_text">
-                <Original>Rescue Missions require Cities, airports or beaches closer than %1m from your HQ.</Original>
+                <Original>Rescue missions require cities, airports or beaches closer than %1m from your HQ.</Original>
```
```suggestion
                <Original>Rescue missions require cities, airports or beaches within %1m of your HQ.</Original>
```
subjective, but this is awkward wording

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2441,7 +2441,7 @@
             </Key>
             <Key ID="STR_cut_text_paradrop_notification_1">
-                <Original>&lt;t color='#d3d9df' size='2'&gt;Commander uses paradrop insertion at some location. You're about to be moved into plane in %1 seconds. Press ESCAPE if you don't want to be moved into airplane. Parachutes will be automatically applied on jump from plane.&lt;/t&gt;</Original>
+                <Original>&lt;t color='#d3d9df' size='2'&gt; The commander has begun a paradrop insertion at a marked location. You're about to be moved into the plane in %1 seconds. Press ESCAPE if you don't want to be moved into the airplane. Parachutes will be automatically applied on jump from plane.&lt;/t&gt;</Original>
```
```suggestion
                <Original>&lt;t color='#d3d9df' size='2'&gt; The commander has begun a paradrop insertion at a marked location. You will be moved into the plane in %1 seconds. Press ESCAPE if you don't want to be moved into the airplane. Parachutes will automatically deploy upon jumping from the plane.&lt;/t&gt;</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2455,7 +2455,7 @@
             </Key>
             <Key ID="STR_cut_text_paradrop_notification_3">
-                <Original>&lt;t color='#d3d9df' size='2'&gt;You have been moved to airplane to attend in paradrop.&lt;/t&gt;</Original>
+                <Original>&lt;t color='#d3d9df' size='2'&gt;You have been moved to the  airplane to attend in the paradrop.&lt;/t&gt;</Original>
```
```suggestion
                <Original>&lt;t color='#d3d9df' size='2'&gt;You have been moved into the airplane to participate in the paradrop.&lt;/t&gt;</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2520,7 +2520,7 @@
             </Key>
             <Key ID="STR_hints_AS_traitor_intel">
-                <Original>Someone found some intel on the traitors family, he will not cause any problems any more!</Original>
+                <Original>Someone found some intel on the traitors family, he will not cause us any problems any more!</Original>
```
```suggestion
                <Original>Someone found some intel on the traitor's family. He will not cause us problems anymore!</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2984,21 +2984,21 @@
             </Key>
             <Key ID="STR_chats_mission_request_big_battle">
-                <Original>There is a big battle around, I don't think the enemy will send any convoy.</Original>
+                <Original>There is a large battle occurring, I don't think the enemy will be sending any convoys.</Original>
```
```suggestion
                <Original>There is a large battle occurring; I don't think the enemy will be sending any convoys.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -4105,7 +4105,7 @@
             </Key>
             <Key ID="STR_notifiers_disband_tip">
-                <Original>Click on any friendly marker to disband and refund some part of its cost.</Original>
+                <Original>Click on any friendly marker to disband it and recieve a partial refund.</Original>
```
```suggestion
                <Original>Click on any friendly marker to disband it and receive a partial refund.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -4519,7 +4519,7 @@
             </Key>
             <Key ID="STR_params_rest_commander_only">
-                <Original>Only commander can order to rest.</Original>
+                <Original>Only commander can order a rest.</Original>
```
```suggestion
                <Original>Only the commander can order a rest.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -4682,7 +4682,7 @@
             </Key>
             <Key ID="STR_garrison_error_no_hr">
-                <Original>You lack enough HR to recruit.</Original>
+                <Original>You lack enough HR to recruit anyone.</Original>
```
```suggestion
                <Original>You don't have enough HR for this recruitment.</Original>
```
Is this message only triggered when attempting to recruit an individual rebel or also when attempting to recruit a squad? Could cause confusion if e.g. you have 3 HR and try to recruit a full infantry squad. The message will state you can't recruit *anyone*. That wouldn't be actually true, it just means you can't recruit that many rebels.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -5325,7 +5325,7 @@
             </Key>
             <Key ID="STR_params_ironManMode">
-                <Original>Iron Man Mode - Petros will have single life and game will end if he dies, save will be erased too</Original>
+                <Original>Iron Man Mode - Petros will have a single life and the game will end if he dies, saves will also be erased.</Original>
```
```suggestion
                <Original>Iron Man Mode - Petros will have a single life. If he dies, the game will end and the save will be erased.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -5755,7 +5755,7 @@
             </Key>
             <Key ID="STR_params_afk_uav">
-                <Original>[Experimental] Should player be considered AFK while controlling a UAV?</Original>
+                <Original>[Experimental] Should a player be considered AFK while controlling a UAV?</Original>
```
```suggestion
                <Original>Should a player be considered AFK while controlling a UAV?</Original>
```

I think we can safely remove all [Experimental] tags (for all languages) in the param name strings since they now fall under a dedicated experimental tab / LB selection.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_medic">
-                <Original>You have selected a Medic role.&lt;br/&gt;&lt;br/&gt;Medics do not have any bonus or penalties, but have the ability to use certain medical items for full health restoration.</Original>
+                <Original>You have selected the Medic role.&lt;br/&gt;&lt;br/&gt;Medics do not have any bonus or penalties, but have the ability to use certain medical items for full health restoration.</Original>
```
I would revert this change, since you can have multiple medics. *The* Medic, to me, implies a single position.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_teamleader">
-                <Original>You have selected a Team Leader role.&lt;br/&gt;&lt;br/&gt;Team Leaders are more lightweight units with increased camouflage capabilities who can hire AI units into their squad.</Original>
+                <Original>You have selected the Team Leader role.&lt;br/&gt;&lt;br/&gt;Team Leaders are more lightweight units with increased camouflage capabilities who can hire AI units into their squad.</Original>
```
I would revert this change, since you can have multiple TLs. *The* Team Leader, to me, implies a single position.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_rifleman">
-                <Original>You have selected a Rifleman role.&lt;br/&gt;&lt;br/&gt;Riflemen have a 25% bonus on carry capacity but are easy to spot and make more noise when they move.</Original>
+                <Original>You have selected the Rifleman role.&lt;br/&gt;&lt;br/&gt;Riflemen have a 25% bonus on carry capacity but are easy to spot and make more noise when they move.</Original>
```
I would revert this change, since you can have multiple riflemen. *The* Rifleman, to me, implies a single position.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_autorifleman">
-                <Original>You have selected an Autorifleman role.&lt;br/&gt;&lt;br/&gt;Autoriflemen have a slight bonus on carry capacity, but make more noise when they move.</Original>
+                <Original>You have selected the Autorifleman role.&lt;br/&gt;&lt;br/&gt;Autoriflemen have a slight bonus on carry capacity, but make more noise when they move.</Original>
```
I would revert this change, since you can have multiple AR. *The* Autorifleman, to me, implies a single position.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_engineer">
-                <Original>You have selected a Engineer role.&lt;br/&gt;&lt;br/&gt;Engineers do not have any bonus or penalties, but have the ability to use repair Kits for vehicle repair and can hack UAV drones.</Original>
+                <Original>You have selected the Engineer role.&lt;br/&gt;&lt;br/&gt;Engineers do not have any bonus or penalties, but have the ability to use repair Kits for vehicle repair and can hack UAVs</Original>
```
```suggestion
                <Original>You have selected an Engineer role.&lt;br/&gt;&lt;br/&gt;Engineers do not have any stat bonuses or penalties, but have the ability to use repair kits for vehicle repair and can hack drones.</Original>
```

Same comment wrt multiple positions for that role, and drones is a more general term (applies to both UAVs and UGVs).

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6047,47 +6047,47 @@
             </Key>
             <Key ID="STR_role_grenadier">
-                <Original>You have selected a Grenadier role.&lt;br/&gt;&lt;br/&gt;Grenadiers have a slight bonus on carry capacity, but are easy to spot.</Original>
+                <Original>You have selected the Grenadier role.&lt;br/&gt;&lt;br/&gt;Grenadiers have a slight bonus on carry capacity, but are easy to spot.</Original>
```
I would revert this change, since you can have multiple grenadiers. *The* Grenadier, to me, implies a single position.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6243,14 +6243,14 @@
             </Key>
             <Key ID="STR_info_bar_undercover_break_reason_garrison_recognition">
-                <Original>Garrison has recognised you!</Original>
+                <Original>The garrison has recognised you!</Original>
```
```suggestion
                <Original>The garrison has recognized you!</Original>
```

DrunkOnSuccess replied:
I'm not 100% on this one, both spellings of 'recognised' are correct. "Recognised" is the British English way of spelling it and "recognized" is the American English spelling.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6271,7 +6271,7 @@
             </Key>
             <Key ID="STR_info_bar_undercover_break_reason_left_wanted">
-                <Original>You left your vehicle and you are still on the Wanted List!</Original>
+                <Original>You left your vehicle and you are still on the wanted list!</Original>
```
In this case, wouldn't "Wanted List" be a proper noun (and thus capitalized) since it refers to a *specific* entity? https://en.wikipedia.org/wiki/Proper_noun

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6969,7 +6969,7 @@
             </Key>
             <Key ID="STR_antistasi_dialogs_flip_vehicle_tooltip">
-                <Original>Flips the vehicle you're pointing at. Requires 3 people to successfully perform. Heavy vehicles requires repair truck in proximity.</Original>
+                <Original>Flips the vehicle you're looking at. requires 3 people to successfully perform. Heavy vehicles requires another vehicle in proximity.</Original>
```
```suggestion
                <Original>Flips the vehicle you're looking at. Requires 3 people to successfully perform. Heavy vehicles require another vehicle in close proximity.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -6997,7 +6997,7 @@
             </Key>
             <Key ID="STR_antistasi_dialogs_put_garage_tooltip">
-                <Original>Vehicle or Static weapon you're looking at will be garaged; interact with Flag to retrieve from garage.</Original>
+                <Original>The vehicle or static weapon you're looking at will be garaged; interact with the flag/garage box to retrieve from garage.</Original>
```
```suggestion
                <Original>The vehicle or static weapon you're looking at will be garaged; interact with the flag or garage box to retrieve it from the garage.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -7477,7 +7477,7 @@
             </Key>
             <Key ID="STR_commander_menu_lva_tooltip">
-                <Original>Plane flies over and drops two-seated light vehicle to the specified position.</Original>
+                <Original>A plane flies over and drops a two-seated light vehicle to the specified position.</Original>
```
```suggestion
                <Original>A plane flies over and drops a two-seated light vehicle at the specified position.</Original>
```

Consistency with your other two changes.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -7505,7 +7505,7 @@
             </Key>
             <Key ID="STR_commander_menu_hmga_tooltip">
-                <Original>Plane flies over and drops crate with HMG to the specified position.</Original>
+                <Original>A plane flies over and drops crate with HMG at the specified position.</Original>
```
```suggestion
                <Original>A plane flies over and drops a crate with HMG at the specified position.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -8564,7 +8564,7 @@
             </Key>
             <Key ID="STR_antistasi_rivals_hint_text">
-                <Original>%1 has started to build their shadow network across the %2.&lt;br/&gt; Their nature is decentralized, their cells are autonomous, their ranks have various specialists with various backgrounds - from deserters and international criminals to mercenaries and paramilitaries. They are trained well and know how to fight insurgency - they will try to disrupt your operations wit ...
+                <Original>%1 has started to build their shadow network across %2.&lt;br/&gt; Their nature is decentralized, their cells are autonomous, their ranks have various specialists with various backgrounds - from deserters and international criminals to mercenaries and paramilitaries. They are trained well and know how to fight insurgencies - they will try to disrupt our operations with d ...
```
```suggestion
                <Original>%1 has started to build their shadow network across %2.&lt;br/&gt; Their nature is decentralized, their cells are autonomous, their ranks have various specialists with various backgrounds - from deserters and international criminals to mercenaries and paramilitaries. They are trained well and know how to fight insurgencies - they will try to disrupt your operations with dirty methods, covert operations and sheer force. Soon they will find you.&lt;br/&gt;&lt;br/&gt; Use new commander menu tab or new tripod map action to see your current progress on eliminating this new threat.</Original>
```

Change from "your" to "our" is inconsistent with the rest of the blurb (...they will find *you*...see *your* current progress...)

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9331,7 +9331,7 @@
             </Key>
             <Key ID="STR_CON_Outpost_resources_compet_desc">
-                <Original>A %1 would be a fine addition to our cause. Go there and capture it before %2. %3 Sent their troups as well</Original>
+                <Original>A %1 would be a fine addition to our cause. Go there and capture it before %2. %3 Sent their troops as well</Original>
```
```suggestion
                <Original>A %1 would be a fine addition to our cause. Go there and capture it before %2. %3 sent their troops as well</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9343,7 +9343,7 @@
             </Key>
             <Key ID="STR_CON_Outpost_controls_compet_desc">
-                <Original>A %1 interferes with our operations in the area. Go there and destroy it before %2. %3 Sent their troups as well</Original>
+                <Original>A %1 interferes with our operations in the area. Go there and destroy it before %2. %3 Sent their troops as well</Original>
```
```suggestion
                <Original>A %1 interferes with our operations in the area. Go there and destroy it before %2. %3 sent their troops as well</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9355,7 +9355,7 @@
             </Key>
             <Key ID="STR_CON_Outpost_outposts_compet_desc">
-                <Original>A %1 is disturbing our operations in the area. Go there and capture it before %2. %3 Sent their troups as well</Original>
+                <Original>A %1 is disturbing our operations in the area. Go there and capture it before %2. %3 Sent their troops as well.</Original>
```
```suggestion
                <Original>A %1 is disturbing our operations in the area. Go there and capture it before %2. %3 sent their troops as well.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9368,7 +9368,7 @@
             </Key>
             <Key ID="STR_CON_MilAdmin_desc">
-                <Original>A Military Administration of Kavala asserts their rule and force on our population, we should deal with it so people will breathe freely. Kill military administrator to seize it. Do it before %2.</Original>
+                <Original>A military administration forces their rule on the local population, we should deal with it so the people can live freely. Free any prisoners you find and kill the military officer stationed there to seize it. Do it before %2.</Original>
```
```suggestion
                <Original>A military administration forces their rule on the local population. We should deal with it so the people can live freely. Free any prisoners you find and kill the military officer stationed there to seize it. Do this before %2.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9452,7 +9452,7 @@
             </Key>
             <Key ID="STR_A3A_Missions_AS_Traitor_task_desc">
-                <Original>A traitor has scheduled a meeting with %1 in %2. Kill him before he provides enough intel to give us trouble. Do this before %3. We don't know exactly where this meeting will happen. You will recognise the building by the nearby Offroad and %1 presence.</Original>
+                <Original>A traitor has scheduled a meeting with the %1 in %2. Kill him before he provides enough intel to give us trouble. Do this before %3. We don't know exactly where this meeting will happen but you will be able to find the building he is in by the nearby offroad and %1 presence.</Original>
```
```suggestion
                <Original>A traitor has scheduled a meeting with the %1 in %2. Kill him before he provides enough intel to give us trouble. Do this before %3. We don't know exactly where this meeting will happen but you will be able to find the building he is in by the nearby offroad vehicle and %1 presence.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9894,7 +9894,7 @@
             </Key>
             <Key ID="STR_A3A_Missions_RES_Informer_task_desc">
-                <Original>%1 forces sweeps %2 in search of our civilian informer with some very valuable data. He's hiding in one of the city's buildings. We have to get him out of there or he will be killed and our intel source will be exposed. He will shoot flare in the sky when we are close to him. Bring him to our HQ. Do this before %3.</Original>
+                <Original>%1 forces sweeps %2 in search of our civilian informant with some very valuable data. He's hiding in one of the town's buildings. We have to get him out of there or he will be killed and our intel source will be exposed. He will shoot flare in the sky when we are close to him. Bring him to our HQ. Do this before %3.</Original>
```
```suggestion
                <Original>%1 forces sweep %2 in search of our civilian informant with some very valuable data. He's hiding in one of the town's buildings. We have to get him out of there, or he will be killed and our intel source will be exposed. He will shoot a flare in the sky when we are close to him. Bring him to our HQ. Do this before %3.</Original>
```

DrunkOnSuccess replied:
Just gotten to the end now, thanks for your help. I tried committing most of your corrections but Github is throwing a hissy so I'm going to have to do all of the changes locally and then push an updated file. Once I've done that I'm going to mark these as resolved, thanks again.

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9880,7 +9880,7 @@
             </Key>
             <Key ID="STR_A3A_Missions_RES_Refugees_task_desc_2">
-                <Original>A group of refugees is hiding in %1 awaiting for evacuation. We have to find them before %2 does it. If not, there will be a certain death for them. Bring them back to HQ. Do this before %3.</Original>
+                <Original>A group of refugees is hiding in %1 awaiting for evacuation. We have to find them before %2 does. If not, it is certain death for them. Bring them back to HQ. Do this before %3.</Original>
```
```suggestion
                <Original>A group of refugees is hiding in %1 waiting for evacuation. We have to find them before %2 does. If not, it is certain death for them. Bring them back to HQ. Do this before %3.</Original>
```

jwoodruff40 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -9826,7 +9826,7 @@
             </Key>
             <Key ID="STR_A3A_Missions_RES_Prisoners_task_desc">
-                <Original>A group of POWs is awaiting execution in %1. We must rescue them before %2. Bring them to HQ.</Original>
+                <Original>A group of POWs are awaiting execution at the %1. We must rescue them before %2. Bring them to HQ.</Original>
```
```suggestion
                <Original>A group of POWs is awaiting execution at the %1. We must rescue them before %2. Bring them to HQ.</Original>
```

the subject is "group", not "POWs"

jwoodruff40 reviewed (commented):
Some of these comments are subjective, some are grammatical errors. Mostly pretty good. Feel free to just mark as resolved on the subjective ones if you don't agree.

DrunkOnSuccess committed:
Update 3

Cleaned up some mistakes and oversights thanks to the help of jwoodruff40. Github wouldn't let me commit the changes he'd already suggested so I've had to do it manually.

DrunkOnSuccess committed:
Update 4

Did one more pass and cleared up some more dialogue.

DrunkOnSuccess committed:
Typos

Ugh.

Silence-112 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -3128,7 +3128,7 @@
             </Key>
             <Key ID="STR_chats_airdrop_pilot_drop_cargo_end">
-                <Original>Cargo has been dropped, I hope it will help the resistance. Good luck, lads!</Original>
+                <Original>Cargo has been dropped, I hope it will help the resistance. Good luck lads!</Original>
```
```suggestion
                <Original>Cargo has been dropped, I hope it will help the resistance. Good luck Comrades!</Original>
```
I think lads is a bit weird here, i'm surprised Socrates used it originally

Silence-112 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -11477,7 +11477,7 @@
             </Key>
             <Key ID="STR_antistasi_actions_talk_with_civ_success_dealer1">
-                <Original>I have some information on a "legitimate" dealer. I hear he's good friends with that weird Irish man.</Original>
+                <Original>I have some information on a "legitimate" dealer. I hear he's good friends with that sketchy Irish man.</Original>
```
```suggestion
                <Original>I have some information on a "legitimate" dealer. I hear he's good friends with that weird Irish man.</Original>
```
Using weird here was intentional so as not to overuse sketchy - the idea being that the Irish man is a rumour that everyone has their own interpretation of

Silence-112 commented on A3A/addons/scrt/Stringtable.xml:
```diff
@@ -2520,7 +2520,7 @@
             </Key>
             <Key ID="STR_hints_AS_traitor_intel">
-                <Original>Someone found some intel on the traitors family, he will not cause any problems any more!</Original>
+                <Original>Someone found some intel on the traitor's family. He will not cause us any problems anymore!</Original>
```
```suggestion
                <Original>Someone found intel on the traitor's family. He will not cause any further problems!</Original>
```

DrunkOnSuccess replied:
All suggestions included in newest update to the file

Silence-112 reviewed (commented):
Generally does improve the strings, I do have a few suggestions

DrunkOnSuccess committed:
Update Stringtable.xml

Silence-112 reviewed (dismissed):
Looks much better

Silence-112 committed:
Merge branch 'unstable' into pr/545
